loom/script/serialize/lsBinWriter.cpp

```diff
@@ -565,7 +565,7 @@ void BinWriter::writeAssembly(const char *sjson, int jsonSize)
     json_error_t jerror;
     json_t       *json = json_loadb(sjson, jsonSize, 0, &jerror);
 
-    lmAssert(json, "Error loading Assembly json: %s\n %s %i\n", jerror.source, jerror.text, jerror.line);
+    lmAssert(json, "Error loading Assembly json: %s\n %s %i\n>>>%s<<<", jerror.source, jerror.text, jerror.line, sjson);
```

Comment:
OK. Can that backfire if we are loading a large JSON chunk with a typo (ie it will blow up trying to dump many megs of JSON)?